h/static/scripts/annotator/plugin/toolbar.coffee

```diff
@@ -63,6 +84,9 @@ class Annotator.Plugin.Toolbar extends Annotator.Plugin
     @buttons.appendTo(list)
     @toolbar.append(list)
 
+    # Hide highlight button.
+    $(@buttons[3]).hide()
```

Comment:
This risks getting broken when we change the order of buttons. It's unnecessarily fragile. I would instead change the `class` key on the button definition above, in line 73, to have `"h-icon-border-color #{HIDE_CLASS}"` and define `HIDE_CLASS` at the top of the module to be `annotator-hide`. Then toggle that class on/off in the `showHighlightButton` method.

I understand that the other methods below here are using indexes into the buttons list in the same manner I've just recommended that you not, but let's try to make problems better not worse. In the future we could change the rest of the places we do this to use attributes to more explicitly filter for the right button.